jetbrains-core/src/software/aws/toolkits/jetbrains/services/federation/OpenArnInConsoleEditorPopupAction.kt

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.federation
+
+import com.intellij.openapi.actionSystem.ActionPlaces
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.ex.util.EditorUtil
+import com.intellij.openapi.project.DumbAwareAction
+
+class OpenArnInConsoleEditorPopupAction : DumbAwareAction() {
```

Comment:
Should this be a DumbAwareAction or just AnAction?

Reply:
intentional; DumbAwareAction is a shorter way of extending AnAction and marking it DumbAware